### PR TITLE
1.0: rewrite_tag_filter: Add example for k8s nested metadata handling

### PR DESCRIPTION
**Pull request by cosmo0920**
Closes https://github.com/fluent/fluentd-docs-gitbook/issues/273

Signed-off-by: Hiroshi Hatake <hatake@clear-code.com>

### CODE REVIEW

File: output/rewrite_tag_filter.md [hunk 1]
```diff
@@ -36,6 +38,59 @@ Sample data:
...
+| kubernetes.information {"kubernetes" : { "namespace_name": "kube-system" }, ... }      | +----> | kube-system.kubernetes.information {"kubernetes" : { "namespace_name": "kube-system" }, ... }         |
+| kubernetes.information {"kubernetes" : { "namespace_name": "default" }, ... }          | +----> | default.kubernetes.information {"kubernetes" : { "namespace_name": "default" }, ... }                 |
+| kubernetes.information {"kubernetes" : { "namespace_name": "kube-public" }, ... }      | +----> | kube-public.kubernetes.information {"kubernetes" : { "namespace_name": "kube-public" }, ... }         |
+| kubernetes.information {"kubernetes" : { "namespace_name": "kube-node-lease" }, ... }  | +----> | kube-node-lease.kubernetes.information {"kubernetes" : { "namespace_name": "kube-node-lease" }, ... } |
++----------------------------------------------------------------------------------------+        +-------------------------------------------------------------------------------------------------------+
+```
+
+These example copnfigurations can process nested attributes like following:
+
```
Comment (kenhys) [inline output/rewrite_tag_filter.md:85]: configurations?

File: output/rewrite_tag_filter.md [hunk 2]
```diff
@@ -36,6 +38,59 @@ Sample data:
...
+```json
+{
+  "kubernetes": {
+    "namespace_name": "kube-system"
+  }
+}
+```
+
```
Comment (kenhys) [inline output/rewrite_tag_filter.md:93]: It may be redundant, but not sure. (because the original record is shown in table)

Comment (cosmo0920) [reply output/rewrite_tag_filter.md]: Hmm....
I decided to remove from the article.
This seems to be redundant. :(

**Review by kenhys — APPROVED** [review]
LGTM

**Pull request merged**